### PR TITLE
Help Center rewrite: Client section

### PR DESCRIPTION
**Pull request by Ephemeralis**
This killed me slightly. Please make sure everything makes sense, etc, et al.

### CODE REVIEW

File: wiki/Help_Center/en.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Having trouble with something? We're here to help! Check out some solutions to c
 
 Usually, these issues resolve themselves within a few hours, and our systems will automatically apply your tag to your account.
 
-If it has been over 24 hours and you still haven't got your supporter tag, please email [support@ppy.sh](mailto://support@ppy.sh) and let us know about your problem.
+If it has been over 24 hours and you still haven't got your supporter tag, please email [support@ppy.sh](mailto:support@ppy.sh) and let us know about your problem.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: haven't got -> have not received

File: wiki/Help_Center/en.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ They may continue to set scores locally and online (though the scores will not s
 
 Restrictions are relatively rare, and are only doled out in circumstances where we believe our Community Rules or Terms of Service have been breached, or an account has been observed to be acting abnormally.
 
-In most cases, the status will be removed automatically after 24 hours. If such time has passed and you're still restricted, you may inquire about the status of your account with the support team by emailing [accounts@ppy.sh](mailto://accounts@ppy.sh).
+In most cases, the status will be removed automatically after 24 hours. If such time has passed and you're still restricted, you may inquire about the status of your account with the support team by emailing [accounts@ppy.sh](mailto:accounts@ppy.sh).
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: you're -> you are

File: wiki/Help_Center/en.md [hunk 3]
```diff
@@ -103,7 +103,7 @@ If you have purchased (or been gifted) a [supporter tag](https://osu.ppy.sh/hom
 
 At any point thereafter, you may purchase a username change to change your name to whatever you wish. However, each change will become progressively more expensive - the second will cost $8 USD, the third $16 USD, and so on. We do this to prevent people from abusing the system to change their names in rapid succession.
 
-If you are only after minor capitalization or grammar/spelling changes, you can email [accounts@ppy.sh](mailto://accounts@ppy.sh) to request a free name change to fix these issues. We won't charge you for these sorts of name changes, but you **MUST** do them through a support email.
+If you are only after minor capitalization or grammar/spelling changes, you can email [accounts@ppy.sh](mailto:accounts@ppy.sh) to request a free name change to fix these issues. We won't charge you for these sorts of name changes, but you **MUST** do them through a support email.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: won't -> will not
`**must**`

File: wiki/Help_Center/en.md [hunk 4]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+
+#### Do you have a default web browser set?
+
+**Under the vast majority of all circumstances, most Windows installations will have a default web browser set.**
+
+To check, try the following steps (under Windows 8 and above):
+
+1. Open the Start menu.
+2. Type `default` into the search bar within the Start menu.
+3. If you see a menu entry titled `Default app settings`, click it. If you don't, expand your search to include `default app settings`.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: don't -> do not or can not

File: wiki/Help_Center/en.md [hunk 5]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
 
-Right click the osu!.exe file in the osu! installation folder and click on Propreties, then go to the Compatibility tab and make sure that the checkbox under Compatibility mode is **not** checked.
+This issue is sometimes caused by an interaction with compatibility mode, and can also be caused by the game simply not finding a default browser to open the web page link with.
 
-If you're not running osu! in compatibility mode and the issue persists, make sure you have a default browser set. For some users that have Chrome as their default browser the issue will still occur for unknown reasons, so try setting Internet Explorer as the default browser and see if your issue persists after that.
+#### Is your game running under Windows compatibility mode?
+
+To check or change if your game is running in compatibility mode under Windows, try the following steps:
+
+1. Open your osu! installation folder and look for `osu!.exe`.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*` ?

File: wiki/Help_Center/en.md [hunk 6]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
-Please ensure that osu! is allowed full access through whichever firewall or anti-virus software you are using - internet connectivity is required to log into our game servers.
+**Ensure that osu! can properly access the internet through any firewall or anti-virus suites you have installed on your computer.**
+
+Here are a few ways of allowing applications to have internet access through several common anti-virus suites:
+
+#### Allowing a program through Windows Firewall
+
+**On most Windows installations, Windows Firewall will be the default firewall software.**
+
+To make sure osu! can access the internet, we'll need to allow it to communicate through Windows Firewall. Here's how to check and add osu! to the program's list:
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `To make sure *osu!* can access the internet, we will need to allow it to communicate through Windoes Firewall. Here is a guide on how to check and add *osu!* to the program list`

File: wiki/Help_Center/en.md [hunk 7]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+* A trace route output log from the command `tracert` for the servers `cho.ppy.sh`, `osu.ppy.sh` and `m1.ppy.sh` (see below for instructions)
+* The `network.log` file from the `Logs` folder in your osu! installation directory
+
+##### Running a trace route to troubleshoot connection issues
+
+**A trace route will help us detect exactly where the problem is between you and our servers.**
+
+You can perform a trace route by opening Command Prompt and running the command `tracert`.
+
+To open the Command Prompt, press and hold the `Windows` key and press `R`, and then type `cmd` and click OK. [For more information, see this Windows knowledgebase article.](https://support.microsoft.com/en-us/kb/314868).
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: ``and click `OK` ``

File: wiki/Help_Center/en.md [hunk 8]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+
+To copy the results of the command, simply click and drag from the bottom of the output all the way to the top so that the information turns white, then right click to instantly copy it to your clipboard.
 
 ### My PC reboots while playing osu!
 
-This is usually caused by your graphics card or CPU overheating, or some other hardware fault.
+**Most of the time, this is due to overheating of one or more of your computer's components, usually your graphics card or CPU.**
 
-To determine the cause more accurately, download a GPU/CPU temperature monitor and keep close watch over your hardware while playing osu!. If your hardware is regularly reaching temperatures of >90C / 194F, you may wish to try enabling the 120FPS/240FPS Frame Limiter in the game's options, as this will significantly reduce the load on both your CPU and GPU.
+To figure out what's causing this, we recommend you use a GPU/CPU temperature monitor and keep an eye on it as you play osu! in the background. A good choice of program to use for this would be [Piriform Speccy](https://www.piriform.com/speccy).
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*`

File: wiki/Help_Center/en.md [hunk 9]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
-If overheating is not the issue, you may wish to investigate your sound card - specifically whether its drivers are up to date or not, along with any peripherals you may have plugged in to your computer.
+#### How can I make osu! run cooler if my computer is overheating?
 
-If none of this helps, please send in a support ticket to [accounts@ppy.sh](mailto://accounts@ppy.sh) with more information on the issue - including your computer's technical specifications, model (where appropriate) and any other information you think may help us discern the cause of your problem.
+**The easiest way is to change your frame limiter settings.**
 
-### I have changed my settings, and now I can't start osu! or it freezes.
+To check what your frame limiter is set to, follow these steps:
+
+1. Open osu!.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `1. Open *osu!*`

File: wiki/Help_Center/en.md [hunk 10]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+
+We recommend using `VSync` in situations where heat is an issue.
+
+You may also wish to look into utilities that allow you to control the speed of your CPU and GPU fans manually.
+
+#### My computer isn't overheating, but the game still causes my computer to reboot!
+
+**The most common cause of forced restarts after overheating issues is sound card problems, followed by malfunctioning peripherals attached to your computer.**
+
+Old sound card drivers or software can cause problems with osu!, but this is not an issue for most people. Try updating your sound card or motherboard drivers.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*`

File: wiki/Help_Center/en.md [hunk 11]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
 
-- Hold down the `Shift` key on your keyboard.
-- While holding `Shift`, double click the osu! icon on your desktop.
-- Continue holding `Shift` until you see the recovery dialog prompt. From there, change your settings to the last known working configuration for you.
+### I have changed my settings, and now I can't start osu! or it freezes.
+
+**Resetting osu! back to its default settings will fix this in most cases.**
 
-If you don't remember what your settings were before, you can use the most optimal configuration below.
+Follow these steps to restore osu! to its default settings:
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `Follow these steps to restore *osu!* to the default settings:`

File: wiki/Help_Center/en.md [hunk 12]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
 ### I can't hold more than a few keys at once while playing osu!mania.
 
-This is due to an issue that most keyboards suffer from called "**ghosting**", where certain key combinations cannot be pressed together. Many newer keyboards can allow up to **6** keys to be held down at once, mostly in random combinations.
+**This is due to a hardware issue common to most keyboards called *ghosting*.**
+
+Ghosting is a somewhat complex topic, but essentially boils down to issues with the way your keyboard is made that prevent it from sending more than a few keys down at once from certain places on the board.
+
+Most newer keyboards will allow up to **6** keys to be held down at once, albeit in random combinations.
+
+You can test your keyboard's ghosting limits by using [Microsoft's Anti-Ghosting utility](http://www.microsoft.com/appliedsciences/content/projects/AntiGhostingExplained.aspx).
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: https

File: wiki/Help_Center/en.md [hunk 13]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+#### What is raw input, and how do I turn it on?
 
-- Mouse: check if your mouse supports a manual setting of DPI (dots per inch). If so, try adjusting it to alter your mouse sensitivity over using the Mouse Sensitivity slider in osu!'s options.
-- Tablet: If you have recently bought a new tablet, and you feel like your mouse is too slow, check out [this screenshot](http://w.ppy.sh/d/dc/Tabarea.png) for instructions on how to change your area.
-- osu! Tablet: osu! tablet is one of tablets that supports raw input - just enable Raw Input in the options and adjust your Mouse Sensitivity like you would a mouse!
+**Raw input forces the game to use a cleaner and more direct means to access the information from your peripherals. This generally increases the accuracy of the output from the device.**
+
+To enable raw input, follow these steps:
+
+1. Open osu!.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `1. Open *osu!*`

File: wiki/Help_Center/en.md [hunk 14]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+Most beatmap packs come in a `.rar` file format, which means they are a compressed archive of beatmap files. You will need to extract them using your utility of choice (we recommend [7-Zip](http://www.7-zip.org/)) to your Songs directory first.
+
+Once all of the `.osz` files within the pack are in the Songs directory, hitting `F5` at the song selection menu will refresh the game's beatmap cache and load your new songs into the game.
 
 ### My game won't update properly!
 
-osu! generally keeps itself up-to-date without any help, but sometimes things can go wrong. This is usually due to unique circumstances on your PC. Here are the steps you should take if updating is failing:
+**This is usually due to issues with your PC, but can very rarely be caused by new updates.**
+
+osu! generally keeps itself up-to-date without any help, but sometimes things can go wrong.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*`

File: wiki/Help_Center/en.md [hunk 15]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+
+We recommend restarting your PC as the first option. This will fix more issues than you expect it to as far as computers are concerned, and osu! is no different.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*`

File: wiki/Help_Center/en.md [hunk 16]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+
+#### How do I check or set my game's release stream?
 
-- Restart your PC. This will solve a lot of issues, even if you think it may not. Please do this first before continuing.
-- Make sure your release stream is set to stable. From the main menu, click Options and find the UPDATES section (near the top). Change your release stream to stable. An image is here.
-- [Try running the updater manually](http://w.ppy.sh/0/01/UpdaterIssues2.png).
-- Force the updater to run by creating a text (`.txt`) document in your osu! folder called "help", then running the game like normal.
+**This can be changed via the in-game options menu.**
+
+1. Open osu!.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `1. Open *osu!*`

File: wiki/Help_Center/en.md [hunk 17]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+2. Enter the `Options` menu or hit `Control` + `O` by default.
+3. Type `release` into the quick-search to jump straight to the option.
+4. Ensure the dropdown says `Stable (Latest)` to receive the latest stable build of the game.
+
+#### How do I force the game to update?
+
+**You can force the game to update via the options menu.**
+
+1. Open osu!.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `1. Open *osu!*`

File: wiki/Help_Center/en.md [hunk 18]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+3. Type `update` into the quick-search to jump straight to the option.
+4. Click the `Run osu! updater` button.
+
+You can also force the game to update by creating a `help.txt` file in osu!'s installation directory. This file does not need to contain anything - it just needs to exist, and the game will automatically force an update the next time it runs
 
 ### I get an error about "font styles" when I try to open the game!
 
+**You have deleted or lost important systme fonts that are bundled with the base Windows installation. This probably isn't the only issue you're noticing!**
+
 If you see this error when you try to open osu! or its updater, it means you deleted or otherwise lost important system fonts that ship with Windows. These fonts are used all over the Windows OS, so you can expect other assorted problems, such as wrong fonts, missing text, and other app crashes throughout Windows.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*`

File: wiki/Help_Center/en.md [hunk 19]
```diff
@@ -239,23 +383,26 @@ See [this topic](http://answers.microsoft.com/en-us/windows/forum/windows_vista-
...
+This helps to identify your own messages from everyone else's. There is no option to disable it at this time.
 
 ### Some of my beatmaps are missing!
 
-There are a few possibilities which can make your maps appear to be missing or deleted, but this is almost always a recoverable scenario. Here are a few things to try:
+**Make sure you're not searching for beatmaps within a certain criteria (such as star rating, approach rate, so on).**
 
-- Make sure you do not have osu! set to only display beatmaps within a certain difficulty range.
-- Press `F5` on the song selection screen for osu! to reprocess your beatmaps. If only some of your beatmaps were imported this should solve your issue.
+If you've just downloaded a map, you may want to try refreshing your beatmap list by pressing `F5` at the song selection screen.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `If you have just downloaded a map`

File: wiki/Help_Center/en.md [hunk 20]
```diff
@@ -298,7 +445,7 @@ You'll no longer see public chat messages from any users on that list, or receiv
 
 #### Blocking all non-friends from sending private messages in-game
 
-If you wish to prevent anyone not on your friendslist from sending you private messages, follow the following steps. **Please note: moderators are not affected by this functionality and can always PM you. If you feel a moderator is harassing you in spite of this, you should contact [our support team](mailto://support@ppy.sh).**
+If you wish to prevent anyone not on your friendslist from sending you private messages, follow the following steps. **Please note: moderators are not affected by this functionality and can always PM you. If you feel a moderator is harassing you in spite of this, you should contact [our support team](mailto:support@ppy.sh).**
 
 1. Click the `Options` menu button in-game or hit `Control` + O on your keyboard
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `` `O` ``

File: wiki/Help_Center/en.md [hunk 21]
```diff
@@ -665,11 +812,11 @@ To determine if your osu! nono keyboard LEDs are faulty, please undertake the fo
...
+**If the "faulty" LED does not work when moved to the alternate slot, but the other LED does,** then the LED is indeed faulty and subject to a replacement. Email [store@ppy.sh](mailto:store@ppy.sh) and request a replacement to be sent, detailing that you followed the instructions in this article.
 
-**If neither LED works after swapping (and they didn't before), then they're probably both broken.** While we cannot conclude whether the mainboard is broken if both do not function, we will be happy to send out replacement LEDs for you to test before we send out a replacement unit entirely. Again, contact [store@ppy.sh](mailto://store@ppy.sh) and detail that you followed the steps in this article.
+**If neither LED works after swapping (and they didn't before), then they're probably both broken.** While we cannot conclude whether the mainboard is broken if both do not function, we will be happy to send out replacement LEDs for you to test before we send out a replacement unit entirely. Again, contact [store@ppy.sh](mailto:store@ppy.sh) and detail that you followed the steps in this article.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: didn't -> did not
they're -> they are

File: wiki/Help_Center/en.md [hunk 22]
```diff
@@ -692,7 +839,7 @@ If the tablet's light does not flash, try using a different USB cable - the ones
 - Screw the grip back onto the pen.
 - Depress the button on the back end (eraser end) of the pen until it clicks.
 
-If your tablet unit is functioning properly, placing the pen near the tablet will make the cursor on your screen move. If this does not occur and you have followed the tablet verification steps listed above this section, then it is possible that your pen is broken or faulty. Contact [store@ppy.sh](mailto://store@ppy.sh) for further assistance, and we'll do everything we can to help you.
+If your tablet unit is functioning properly, placing the pen near the tablet will make the cursor on your screen move. If this does not occur and you have followed the tablet verification steps listed above this section, then it is possible that your pen is broken or faulty. Contact [store@ppy.sh](mailto:store@ppy.sh) for further assistance, and we'll do everything we can to help you.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: we'll -> we will

File: wiki/Help_Center/en.md [hunk 23]
```diff
@@ -712,7 +859,7 @@ Right click on your Start button and choose "Disk Management"
 
 **If it comes up but is unrecognised, you will need to partition it from scratch.** Please follow [this guide](https://tails.boum.org/doc/first_steps/reset/windows/index.en.html) for details on how to do that. Once partitioned appropriately, simply follow the step above this one to format it properly, then place the files from goodies.zip into the drive, and you're done!
 
-If you encounter any issues, please send an email to [store@ppy.sh](mailto://store@ppy.sh) and we'd be happy to help you out.
+If you encounter any issues, please send an email to [store@ppy.sh](mailto:store@ppy.sh) and we'd be happy to help you out.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: we'd -> we would

File: wiki/Help_Center/en.md [hunk 24]
```diff
@@ -739,6 +886,6 @@ This usually happens when there are networking issues between you and our server
 - Make sure that you typed your password correctly. If you're unsure of what your password is, you can use the <https://osu.ppy.sh/p/forgot> page to recover access to your account.
 - If you are using any proxies or VPNs, disable them before attempting to connect again.
 - Hold `Shift` while starting osu! and change to the `Stable (fallback)` release stream.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*`

File: wiki/Help_Center/en.md [hunk 25]
```diff
@@ -739,6 +886,6 @@ This usually happens when there are networking issues between you and our server
...
-- If the error persists, change to the `Cutting Edge` release stream. Let osu! run for a few minutes, then contact [support@ppy.sh](mailto://support@ppy.sh) and provide the `network.log` file that you can find in the Logs folder.
+- If the error persists, change to the `Cutting Edge` release stream. Let osu! run for a few minutes, then contact [support@ppy.sh](mailto:support@ppy.sh) and provide the `network.log` file that you can find in the Logs folder.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `*osu!*`

File: wiki/Help_Center/en.md [hunk 26]
```diff
@@ -44,7 +44,7 @@ We'll never display this information to anyone else, and we encourage you to bla
 
 **Please use [the password recovery page](https://osu.ppy.sh/p/forgot) to reclaim your account automatically, provided that you still have access to the email you registered with.**
 
-If this does not work for you (or you don't have access to your registration email), please send an email to our [support team](mailto://accounts@ppy.sh) with as much information as you can think of to help us acknowledge you as the owner of your account.
+If this does not work for you (or you don't have access to your registration email), please send an email to our [support team](mailto:accounts@ppy.sh) with as much information as you can think of to help us acknowledge you as the owner of your account.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: don't -> do not

File: wiki/Help_Center/en.md [hunk 27]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
 
 ### osu! freezes when I click a link or multiplayer map!
 
-Make sure you're not running osu! in compatibility mode through the osu!.exe propreties and make sure that you have a default browser set.
+**Make sure you're not running osu! in Windows compatibility mode, and that you also have a default web browser defined in your system settings.**
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: you're -> you are

File: wiki/Help_Center/en.md [hunk 28]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+**Under the vast majority of all circumstances, most Windows installations will have a default web browser set.**
+
+To check, try the following steps (under Windows 8 and above):
+
+1. Open the Start menu.
+2. Type `default` into the search bar within the Start menu.
+3. If you see a menu entry titled `Default app settings`, click it. If you don't, expand your search to include `default app settings`.
+4. Scroll down to the `Web browser` heading and check to make sure that a browser which you have installed is properly set.
 
 ### I can't login or connect to the game at all!
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: can't -> can not

File: wiki/Help_Center/en.md [hunk 29]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+
+#### Allowing a program through McAfee
+
+Please consult McAffee's [official knowledgebase entry on the topic](https://service.mcafee.com/webcenter/portal/cp/home/articleview?articleId=TS100813).
 
-Even if your firewall policy allows other games through without adding exceptions, try adding one for osu! - it'll probably help!
+#### I've added osu! to my firewall but I still can't connect to the game!
 
-If you are still having issues connecting to our game servers, please submit a support ticket by sending an e-mail to [accounts@ppy.sh](mailto://accounts@ppy.sh) containing a trace route output log to the following addresses: *cho.ppy.sh*, *osu.ppy.sh* & *m1.ppy.sh*.
+**We'll need extra information from you to help sort this out.**
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: We'll -> We will

File: wiki/Help_Center/en.md [hunk 30]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+
+**A trace route will help us detect exactly where the problem is between you and our servers.**
+
+You can perform a trace route by opening Command Prompt and running the command `tracert`.
+
+To open the Command Prompt, press and hold the `Windows` key and press `R`, and then type `cmd` and click OK. [For more information, see this Windows knowledgebase article.](https://support.microsoft.com/en-us/kb/314868).
+
+You can then type the trace route command into the console. For example, the command `tracert cho.ppy.sh` will track how your computer connects to the game server over the internet. Tracing routes takes a little time, sometimes upwards of a minute or more depending on your connection.
+
+We'll need the results from a trace route for command for the following servers: `cho.ppy.sh`, `osu.ppy,sh` and `m1.ppy.sh`.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: We'll -> We will

File: wiki/Help_Center/en.md (hunk 8 above)
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: what's -> what is

File: wiki/Help_Center/en.md [hunk 31]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
 
-If you don't remember what your settings were before, you can use the most optimal configuration below.
+Follow these steps to restore osu! to its default settings:
 
-Optimal Configuration:
+1. Hold down the `Shift` key on your keyboard.
+2. While holding `Shift`, double click the osu! icon on your desktop.
+3. Continue holding `Shift` until you see the osu! recovery dialog prompt.
+4. When the `osu! configuration` dialog opens, click the `reset settings` button.
+5. If you still aren't having any luck, open the dialog again and click `repair osu!`.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: aren't -> are not

File: wiki/Help_Center/en.md [hunk 32]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+While the Ranking Criteria for osu!mania tries to alleviate the issue of ghosting by not allowing more than 6 notes to be active at any one time, you may encounter this issue if your keyboard is older.
+
+The only true solution is to purchase a keyboard that can support more concurrent keypresses, in that case. You may wish to look for a feature known as *n-key rollover* (or *NKRO*). NKRO keyboards do not have issues with ghosting for the most part, but are usually expensive.
+
+### My cursor goes crazy whenever I try to play osu!
+
+**The most common cause of this is having your in-game mouse sensitivity set too high.**
 
-There are applications available that will help you determine the maximum number of concurrent keypresses that your keyboard can handle. [Check this link for one](http://www.microsoft.com/appliedsciences/content/projects/AntiGhostingExplained.aspx)!
+Jumpy, erratic movement is a prime example of this, particularly if you're using a tablet.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: you're -> you are

File: wiki/Help_Center/en.md [hunk 33]
```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
...
+1. Open osu!.
+2. Enter the `Options` menu or hit `Control` + `O` by default.
+3. Type `update` into the quick-search to jump straight to the option.
+4. Click the `Run osu! updater` button.
+
+You can also force the game to update by creating a `help.txt` file in osu!'s installation directory. This file does not need to contain anything - it just needs to exist, and the game will automatically force an update the next time it runs
 
 ### I get an error about "font styles" when I try to open the game!
 
+**You have deleted or lost important systme fonts that are bundled with the base Windows installation. This probably isn't the only issue you're noticing!**
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: `This probably is not the only issue you are noticing`

File: wiki/Help_Center/en.md [hunk 34]
```diff
@@ -239,23 +383,26 @@ See [this topic](http://answers.microsoft.com/en-us/windows/forum/windows_vista-
...
 
-Don't worry! You are visible to others as yellow, but on your own screen your username will always be white. This is so you can easily identify your own messages from everyone else's.
+**You will always appear white to yourself on the in-game client. Other users will see your glorious yellowness, though!**
+
+This helps to identify your own messages from everyone else's. There is no option to disable it at this time.
 
 ### Some of my beatmaps are missing!
 
-There are a few possibilities which can make your maps appear to be missing or deleted, but this is almost always a recoverable scenario. Here are a few things to try:
+**Make sure you're not searching for beatmaps within a certain criteria (such as star rating, approach rate, so on).**
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: you're -> you are

File: wiki/Help_Center/en.md [hunk 35]
```diff
@@ -239,23 +383,26 @@ See [this topic](http://answers.microsoft.com/en-us/windows/forum/windows_vista-
...
 
-- Make sure you do not have osu! set to only display beatmaps within a certain difficulty range.
-- Press `F5` on the song selection screen for osu! to reprocess your beatmaps. If only some of your beatmaps were imported this should solve your issue.
+If you've just downloaded a map, you may want to try refreshing your beatmap list by pressing `F5` at the song selection screen.
 
 ### I receive some sort of server-side error when trying to play online!
 
-Are you using an open connection, such as a proxy, VPN or university campus internet?
+**This can happen if you're using a shared internet connection, a proxy, or VPN.**
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: you're -> you are

File: wiki/Help_Center/en.md [hunk 36]
```diff
@@ -272,7 +419,7 @@ In rare circumstances, support tickets from chronically offending users may be d
 
 **This can happen if your user page had inappropriate content on it as defined by our [community rules](/wiki/Rules).**
 
-In such cases, you may open up a ticket with the support team by emailing [accounts@ppy.sh](mailto://accounts@ppy.sh) and discussing your situation with them.
+In such cases, you may open up a ticket with the support team by emailing [accounts@ppy.sh](mailto:accounts@ppy.sh) and discussing your situation with them.
 
 Most of time, you'll be allowed to access and alter your user page again, but users who abuse this amnesty may find themselves unable to edit their userpage at all, so be careful!
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: you'll -> you will

File: wiki/Help_Center/en.md [hunk 37]
```diff
@@ -739,6 +886,6 @@ This usually happens when there are networking issues between you and our server
 - Make sure that you typed your password correctly. If you're unsure of what your password is, you can use the <https://osu.ppy.sh/p/forgot> page to recover access to your account.
```
Comment (EpicSupermind) [inline wiki/Help_Center/en.md]: you're -> you are

**Comment by Ephemeralis** [comment]
I've chosen not to bother changing contractions in question headers, because most people ctrl-f'ing for issues aren't going to type "MY OSU WILL NOT CONNECT", they're gonna look for "wont connect". Question answer subheaders have been expanded though, as well as the rest.

Didn't touch troubleshooting or other sections since they're due to be rewritten anyway.

**Pull request merged**